components/browser/session-storage/src/main/java/mozilla/components/browser/session/storage/RecoverableBrowserState.kt

```diff
@@ -11,8 +11,10 @@ import mozilla.components.browser.state.state.recover.RecoverableTab
  *
  * @param tabs The list of restored tabs.
  * @param selectedTabId The ID of the selected tab in [tabs]. Or `null` if no selection was restored.
+ * @param inProgressMediaTabId The ID of the last tab known to have played media.
  */
 data class RecoverableBrowserState(
     val tabs: List<RecoverableTab>,
-    val selectedTabId: String?
+    val selectedTabId: String?,
+    val inProgressMediaTabId: String?
```

Comment:
See comment below for an alternative way to add this state, similar to `lastAccess` which we persist here as well.